ansible/roles/configure_ib/tasks/main.yml

```diff
@@ -44,7 +33,18 @@
     create: yes
   with_items: "{{ ib_modules }}"
   become: yes
-  notify: Reset interface
+  notify:
+    - Unload kernel modules
+    - Load kernel modules
+
+- name: Set selinux to permissive
+  selinux:
+    state: permissive
+    policy: targeted
+  become: yes
+  notify:
+    - Unload kernel modules
+    - Load kernel modules
```

Comment:
If the SELinux mode is changed, I think a node reboot is required.  Have you seen that?

Reply:
Appears to work without a reboot but the change persists through reboot because SELinux mode is set to permissive :)

Reply:
The problem we were seeing before was that the change was not persisting through reboot if selinux mode was enforcing

Reply:
You can change it to permissive without a reboot, but not disable IIRC.